core/templates/components/button-directives/social-buttons.component.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css">
```

Comment:
Why do we need to import Fontawesome? Seems like irrelevant changes, can you please remove it if it's not needed? (Ditto elsewhere)

Reply:
Same here also changes related to pr #19056